src/e_uadk.c

```diff
@@ -60,33 +65,42 @@ static int uadk_finish(ENGINE *e)
 static int bind_fn(ENGINE *e, const char *id)
 {
 	struct uacce_dev_list *list;
+	const char *acc_device = "hisi_acc";
+	static int loaded;
 
 	if (id && (strcmp(id, engine_uadk_id) != 0)) {
 		fprintf(stderr, "wrong engine id\n");
 		return 0;
 	}
 
+	if (loaded) {
+		fprintf(stderr, "\n%s\n", __func__);
+		return 0;
+	}
+	loaded = 1;
+
 	if (!ENGINE_set_id(e, engine_uadk_id) ||
 	    !ENGINE_set_destroy_function(e, uadk_destroy) ||
-	    !ENGINE_set_init_function(e, uadk_init) ||
+		!ENGINE_set_init_function(e, uadk_init) ||
```

Comment:
why change this?

Reply:
Emmm, I'm sorry, that is an unintentional tab mistake, I'll recover it.